src/components/Card/index.js

```diff
@@ -1,206 +1,16 @@
 import './card.scss'
 
-import { SbMenu, SbMenuButton, SbMenuList } from '../Menu'
-import SbLoading from '../Loading'
-
-// @vue/component
-const SbCardHeader = {
-  name: 'SbCardHeader',
-
-  props: {
-    as: {
-      type: String,
-      default: 'span',
-    },
-    options: {
-      type: Array,
-      default: null,
-    },
-    title: {
-      type: String,
-      default: null,
-    },
-  },
-
-  render(h) {
-    const renderTitle = () => {
-      if (this.title) {
-        return h(
-          this.as,
-          {
-            staticClass: 'sb-card__title',
-          },
-          this.title
-        )
-      }
-
-      return this.$slots.default
-    }
-
-    return h(
-      'div',
-      {
-        staticClass: 'sb-card__header',
-        attrs: {
-          ...this.$attrs,
-        },
-      },
-      [
-        renderTitle(),
-        this.options &&
-          h(SbCardOptions, {
-            props: {
-              options: this.options,
-            },
-          }),
-      ]
-    )
-  },
-}
-
-// @vue/component
-const SbCardOptions = {
-  name: 'SbCardOptions',
-
-  props: {
-    options: {
-      type: Array,
-      default: () => [],
-    },
-  },
-
-  render(h) {
-    return h(
-      'div',
-      {
-        staticClass: 'sb-card__options',
-      },
-      [
-        h(SbMenu, [
-          h(SbMenuButton, {
-            props: {
-              iconName: 'menu-horizontal',
-              hasIconOnly: true,
-              variant: 'ghost',
```

Comment:
This variant got lost, it should probably be `tertiary` now, because we changed from 'ghost' to 'tertiary' 

Production Card:
<img width="980" alt="image" src="https://user-images.githubusercontent.com/11278408/183368075-6eda9b72-ab4f-4bd1-9136-69911be00285.png">

Your Card: 
<img width="984" alt="image" src="https://user-images.githubusercontent.com/11278408/183368160-8be1bad2-0c97-4f1c-9273-186843a0ec19.png">